src/test/java/visualizations/BarChartTest.java

```diff
@@ -0,0 +1,50 @@
+package visualizations;
```

Comment:
I don't think currently we need these tests. Aren't these covered by the examples?

Reply:
agree. removed the tests.